package.json

```diff
@@ -41,9 +43,13 @@
   "scripts": {
     "lint": "pnpm -r lint",
     "lint:check": "pnpm -r lint:check",
-    "postinstall": "pnpm run pre-build:contracts-sdk && pnpm run build:contracts && pnpm run build:contracts-sdk",
+    "postinstall": "pnpm run postinstallEvm && pnpm run postinstallTezos",
+    "postinstallEvm": "pnpm run pre-build:contracts-sdk && pnpm run build:contracts && pnpm run build:contracts-sdk",
+    "postinstallTezos": "cd tezos-sig-gating-contracts && make install && make compile && cp compiled/extended_gated_nftminter.json src/compiledcontracts/ && cd .. && pnpm run build:tezos-contracts-sdk",
```

Comment:
Do we always need to build the docker image after executing pnpm install? If yes, we can leave it as it

Reply:
the docker image is the LIGO compiler. It is necessary to compile contracts written in Ligo. Maybe there is a place where to store docker images ? in order to download it only once ?

Reply:
We can leave it as it is, it will be stored in our local machines docker - for CI it will be fine if it's always downloaded